tests/x11/rmt/rmt_feature.pm

```diff
@@ -0,0 +1,97 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Add rmt configuration test and basic configuration via
+#    rmt-wizard, test enable/disable products/repo, test rmt sync
+#    rmt mirror, test import SMT data to RMT
+# Maintainer: Yutao wang <yuwang@suse.com>
+
+use strict;
+use warnings;
+use testapi;
+use base 'x11test';
+use repo_tools;
+use utils;
+use x11utils 'turn_off_gnome_screensaver';
+
+sub run {
+    x11_start_program('xterm -geometry 150x35+5+5', target_match => 'xterm');
+    # Avoid blank screen since smt sync needs time
+    turn_off_gnome_screensaver;
+    become_root;
+    rmt_wizard();
+    # sync from SCC
+    rmt_sync;
+    # enable all modules of products at one arch
+    my $pro = get_var("PRODUCT_ALLM");
+    assert_script_run("rmt-cli p e $pro --all-modules");
+    assert_script_run("rmt-cli product list | grep $pro");
+    # enable all modules of products at four arch
+    my $pro1 = get_var("PRODUCT_ALLMA");
```

Comment:
i suggest we combine two new setting to one, such as RMT_TEST_PRODUCT=15.2

Reply:
As discussed in the meeting. Not change now.